### PR TITLE
Sync kubelet with kubeadm (and cleanup)

### LINKED ISSUE #3355: Sync Kubelet config with kubeadm
**Component**:

'salt', 'kubernetes'

**Why this is needed**:

The kubelet config do not get updated since a long time we should keep this one in sync with what is done by kubeadm

**What should be done**:

- Add kubelet config "support" in `get-kubeadm-manifests` tool
- Update kubelet config to match (more or less) kubeadm one

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
**Component**:

'salt', 'kubelet'

**Context**: 

Sync the kubelet config we deploy with what is done in kubeadm so that future Kubernetes version bump will be simpler

**Summary**:

- Add kubelet config in `get-kubeadm-manifest` tool
- Sync kubelet config

NOTE: The only real changes here in the kubelet config is `nodeStatusReportFrequency` that was set to `1m0s` and now is set to
`0s` (so we use the default which is `5m`)

See the commit message for more details

---

Fixes: #3355

### CODE REVIEW

File: salt/metalk8s/kubernetes/kubelet/standalone.sls [hunk 1]
```diff
@@ -40,78 +40,52 @@ Create kubelet config file:
...
           mode: Webhook
           webhook:
-            cacheAuthorizedTTL: 5m0s
-            cacheUnauthorizedTTL: 30s
-        cgroupDriver: cgroupfs
-        cgroupsPerQOS: true
+            cacheAuthorizedTTL: 0s
+            cacheUnauthorizedTTL: 0s
+        # NOTE: we use cgroupfs for the moment
+        # cgroupDriver: systemd
```
Comment (TeddyAndrieux) [inline salt/metalk8s/kubernetes/kubelet/standalone.sls:59]: This change will be done in another PR, 
I tried and it does not work as-is (maybe we need to do some changes on containerd and/or use a newer containerd version, not sure yet)

Comment (gdemonet) [reply salt/metalk8s/kubernetes/kubelet/standalone.sls:59]: Do we have a ticket to track this topic? Could make sense, at least to publicly discuss the change and potential impact.

Comment (TeddyAndrieux) [reply salt/metalk8s/kubernetes/kubelet/standalone.sls:59]: Not yet, but right, I can create one

Comment (TeddyAndrieux) [reply salt/metalk8s/kubernetes/kubelet/standalone.sls:59]: #3366

**Review by alexandre-allard — APPROVED** [review]
